spec/chartmogul/import/customer_spec.rb

```diff
@@ -86,6 +86,23 @@
     end
   end
 
+  describe '.find_by_external_id', vcr: true do
+    let(:customer) { ChartMogul::Import::Customer.create!(attrs) }
```

Comment:
There's no need to create the customer here - VCR will replay the request, so you can simply fetch an existing customer.

Remember to regenerate the API keys in your test ChartMogul account when you do this, as they're accessible in the .yml fixtures.